_posts/2023-02-07-homebrew-4.0.0.md

```diff
@@ -0,0 +1,53 @@
+- [The `homebrew/ubuntu16.04:master` image has been deprecated](https://github.com/Homebrew/brew/pull/13819).
+- [Various major release deprecations and disables](https://github.com/Homebrew/brew/pull/14382).
+
+
+Other changes since 3.6.0 I'd like to highlight are the following:
+
+- [`brew test` sets `PYTHONDONTWRITEBYTECODE` to improve performance.](https://github.com/Homebrew/brew/pull/14338).
+- [VSCode devcontainers are available in the Homebrew/brew repository for use with e.g. GitHub Codespaces](https://github.com/Homebrew/brew/pull/14027).
+- [`brew install --adopt` allows taking ownership of existing installed casks](https://github.com/Homebrew/brew/pull/14033).
+- [`brew install --dry-run` allows viewing what `brew install` will do before it is run](https://github.com/Homebrew/brew/pull/13706).
+- [`brew docs` opens](https://github.com/Homebrew/brew/pull/13844) [docs.brew.sh](https://docs.brew.sh).
+- [`--cask` is never required on Linux](https://github.com/Homebrew/brew/pull/14206).
+- [`service do` blocks allow defining a `run` command per platform](https://github.com/Homebrew/brew/pull/14371).
+- [`brew install` uses the local cache while installing dependencies from `pip`](https://github.com/Homebrew/brew/pull/14239).
+- [`brew doctor` no longer complains about BitDefender](https://github.com/Homebrew/brew/pull/14300).
+- [`brew install` will also suggest casks rather than just formula when it fails to find the requested package](https://github.com/Homebrew/brew/pull/14240).
+- [`brew readall` simulates all architecture and OS configurations for better reliability](https://github.com/Homebrew/brew/pull/13990).
+- [`git` partial clones with sparse checkouts are supported when downloading from `git`](https://github.com/Homebrew/brew/pull/14035).
+- [GCC 12 is used for runtime libraries to match Ubuntu 22.04, where we build our bottles](https://github.com/Homebrew/brew/pull/13882).
+- [`brew doctor --quiet` prints no output on success](https://github.com/Homebrew/brew/pull/14515).
+- [`brew gist-logs` better detects missing permissions](https://github.com/Homebrew/brew/pull/14523).
+- [`brew update` uses the GitHub API token if available to avoid hitting rate-limits](https://github.com/Homebrew/brew/pull/14349).
+- [We have discontinued WSL 1 support and recommend WSL 2 instead](https://github.com/Homebrew/brew/pull/14000).
+- [`brew fetch` and `brew install` can automatically determine mirrors for `glibc-bootstrap` and PyPI resources](https://github.com/Homebrew/brew/pull/13875).
+
+Finally:
+
+- Homebrew is now on Mastodon: [@homebrew@fosstodon.org](https://fosstodon.org/@homebrew).
+- [Our governance documents have been updated as part of our AGM](https://github.com/Homebrew/brew/pull/14482).
```

Comment:
```suggestion
- [Our governance documents have been updated as part of our AGM](https://github.com/Homebrew/brew/pull/14482). Read the new editions of [Homebrew Governance](https://docs.brew.sh/Homebrew-Governance) and [New Maintainer Checklist](https://docs.brew.sh/New-Maintainer-Checklist) if you're interested.
```

Reply:
Nah, I think that's probably overkill. Even including it in these notes is tenuous as most users really don't care 😁